sdk/spring/azure-spring-boot-starter-active-directory/resource/source-file/resource-server.drawio

```diff
@@ -1 +1 @@
\ No newline at end of file
+<mxfile host="Electron" modified="2021-07-21T08:39:10.961Z" agent="5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) draw.io/14.6.13 Chrome/89.0.4389.128 Electron/12.0.7 Safari/537.36" etag="vzsj8qLHzlCp0FfOUCu0" version="14.6.13" type="device"><diagram id="kgpKYQtTHZ0yAKxKKP6v" name="Page-1">7Vpbc9soGP01mtl9sEfoZuvRlzg70zTtJN0m6RuWsE2NhRbhW3/9goQsdIvtjZM2u5uHRBzQB/quB4hhj1a7awbjxUcaImJYZrgz7LFhWf2+J35LYJ8BLgAZMGc4zCCzAO7xD5SBIEfXOESJwjKIU0o4jstgQKMIBbyEQcbotjxsRklYAmI4RzXgPoCkjj7gkC/UZ7lmgf+B8HyRzwxM1bOC+WAFJAsY0q0G2VeGPWKU8uxptRshInVX1sukpfewMIYifsoL/U8QfmC3pk+fvo4ff/h0sxx0nEzKBpK1+mC1WL7PNcDoOgqRFGIa9nC7wBzdxzCQvVthcoEt+IqIFhCPIUwW6di88RlyjliUIpYp0fq61adsEONop0HqO64RXSHO9mKI6rX6vewV5VSOWvW2sBBw1JCFZh3H6ivPUF4xP4guFCcelO6a9WhH6+Fkx/xgf2PG0aeHLxa87oCaHh/QVACDOCY4gBzTSLQoM6SRREjYQCrzDiV0LTRpmfeIic+v6V74TCwf1ytyg2eI4Ei0hjFiWCxbjLfHRMGfC2woYoFDgTFlh4ASAuMET1OxcmKGgjVL8AaJFWQhl6IHS8u3Dv4quxLO6PIQAbJ7hgkZUUJZulJ7MhkOhxcyL8hDRpm3b9bNC5y6ee184MXNax0Pk5jiiKfzukPDHVfMRBlf0DmNINENVQ6tIwqvK/ZZTzxZ265VUrbXoGyzQdfma+nabQklqIfSu42TPhgNriaXiRM7r6XPxInZZLtXixPv/cSJe662HfenxQl1+Z+b2w+7p360cO2P+/HtPerkGVDTLQoFd1HNQpNXBVrRZDHmhtJY6e874nyviBhcc1ou8EKpbP+oN56ksK7T83NgvFPis9Zeb2lWTcGZCEE1FzjfePJ7nzUdQ0QkjE2ZyzVZQr36WTqnlhrLJgc9sytqU/HjlQVyyOaIKxk6Ezsi1qnIEaSABagmZ8AY3GvDVCS1rv4Q+5V52pZVG98rEUrxkK2gcNmDpl/gxb1ayvgKCQ4hl7QIBgFKkpTzL5FkUFOpgIBAvEo0HkXwEmXM6q+1pNNDuA6LhmGNSp1YSmzrjKaz9k60i4tGtzH8buBUbIDECgieS9ZL0EwOlmlGFC8yUPAKh2EWkLKwwKLi1NPj8RB5Nj3UEtxhr6RmNfTtSFPi65hdz7eVj74wouwycQcVAXQ2SxA3qqnxPD9rzBh1N7tDfM2kTyWyyIucKDapkMOaVaumm1LO6SpNaOFA7jBlso2Fd1Z2P6Jbma2fug7mj3kmFM9Z1vTdEwjEUfuDt8mGoG+XjGf7btfXf/5ZNrRd7xyxLcnxUn7Sr/nJIM9BIlQ5wwFPXYWhbCGJMbKMgb6pozOtO+tNmnd3p3jWlNBg+a6cpLydAL3eSfY76hY1QS3+danCVE8YwsZE5vIkhlHJkHlFmMJgOU8ZVifIGP9AKnQ+/S0t+bKWWDKPygfg+b+ndjKl6TrZ7kIOByDepR1FEfLm8i/I5xefky0hw2tuddaBDZH1anhYuLZTcQZ9bzJp2ulfjQb+wQvznoime66THfGM4wAXVDxB5SGdequ41ak3uMBhT6NrHE70/t3M+40yhu1V8r/tlVh2ReDJZaUqFlhd02wn7xci3U5lTy6mfXaZtfF2iaS/Dum26seVwzUmsq4Jv1tQhjlGyel0OwnidtLMKEFJiTbr1fLLIuX1SC4l4/wxYitB0TGNfhbDFtk1wNH8TqUSs4Bu0rnGwDudh+fJ4tfi4aDCwyuHFBfh4c3qaLpxeFldNbXSamrV1TLbC2ytutr/V1eVgPIbuzeoro1XUvXz37ucTLfek4hv5WXNN6tP07WC8iwSCA2mZ42teaTJ1GVneKPLL9/plg+QnAaL2VY+SDeadYHDSOYx59v36Jtvzid21Hmytj5quP/6JaLa+q9G9eF4WrmIld98Xj6qRbO4xM7KRPGfAPbV3w==</diagram></mxfile>
```

Comment:
![image](https://user-images.githubusercontent.com/13167207/126458910-78e8732a-247f-4089-ab92-c2aad3916152.png)

Reply:
Look good for me @chenrujun